cmd/kubemark/hollow-node.go

```diff
@@ -97,9 +114,6 @@ func (c *hollowNodeConfig) createClientConfigFromFile() (*restclient.Config, err
 	if err != nil {
 		return nil, fmt.Errorf("error while creating kubeconfig: %v", err)
 	}
-	config.ContentType = c.ContentType
```

Comment:
Any reason why you're moving these lines? If possible, I'd prefer not touching this.